source/api/presentation/3.0/index.md

```diff
@@ -571,6 +571,29 @@ The value _MUST_ be an array of JSON objects. Each object _MUST_ have the `id` a
   }]}
 ```
 
+For cross-version consistency, this specification defines the following types for backwards compatibility with other IIIF APIs. Future versions of these APIs will define their own types.
+
+| Value          | Specification |
+| -------------- | ------------- |
+| ImageService1  | [Image API version 1][]  |
+| ImageService2  | [Image API version 2][]  |
+| SearchService1 | [Search API version 1][] |
```

Comment:
Why the empty links?

Reply:
Ahh... assistance please in getting the new link tricks to work?

Reply:
What's the trick? Something related to https://github.com/IIIF/api/pull/1261?